cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -73,11 +73,26 @@ func startMizarServiceController(ctx *ControllerContext, grpcHost string) (http.
 	klog.V(2).Infof("Starting %v", controllerName)
 
 	svcKubeconfigs := ctx.ClientBuilder.ConfigOrDie(controllerName)
+	for _, svcKubeconfig := range svcKubeconfigs.GetAllConfigs() {
+		svcKubeconfig.QPS *= 20
```

Comment:
What is the original value? *20 seems too aggressive

Reply:
We do this multiplication for namespace controller and tenant controller because they are very chatty during nested deletion scenario.  I don't think we need to multiply it for regular object controllers.